doc/md/cloud/getting-started.md

```diff
@@ -0,0 +1,220 @@
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Discord from '../../website/src/assets/icons/discord-white.svg'
+
+[Ariga Cloud](https://ariga.cloud) is an online platform that allows developers to view their Atlas projects and track
+their [Atlas GitHub Action](/integrations/github-actions) CI runs. The cloud platform gives full visibility into your
+Atlas schema, by displaying an entity relation diagram (ERD) that shows the schema changes, as well as the entire
+database schema.
+
+### Signing Up
+
+1. To get started with [Ariga Cloud](https://ariga.cloud/), create an account by clicking 'sign up' on the homepage.
+2. In the sign up screen, enter your work email and choose a password.
+3. Next, you will receive an email to verify your account. From your email, [Ariga Cloud](https://ariga.cloud/)
+will open in a new tab, and you will need to sign in to access your account.
+4. Once signed in, you will be prompted to create an organization. After creating the organization, you will be able
+to invite team members to join it. Choose a meaningful name for the organization, as it will also be your subdomain.
+For example, "Acme Corp" will be available at "acme-corp.ariga.cloud".
+
+:::note
+Currently, the system only allows users to sign up for **one** organization. If you wish to create multiple
+organizations under the same user, you can create a [task-specific email address](https://support.google.com/a/users/answer/9308648?hl=en)
+(for Google users only). Create multiple emails that all link back to your regular address by adding a plus sign and
+any word before the @ sign in your address.
+:::
+
+### Connecting to the Atlas GitHub action
+At first you will notice that your projects and CI runs pages are empty. In order to connect the organization
+to your GitHub repository, you will need to setup the Atlas GitHub action on your repository by following these steps:
+
+:::note
+If you already have the Atlas GitHub action set up, you may skip step 4. In step 5, only add
+`ariga-token: ${{ secrets.ARIGA_TOKEN }}` to your yaml file.
+:::
+
+1. From the Settings page, generate an access token under 'Tokens'.
+2. On your GitHub repo, under the 'Settings' section, click on 'Secrets' > 'Actions' to create a new repository secret.
+3. Name your secret (for example, ARIGA_TOKEN) and paste the generated token from step 2.
+4. Install the Atlas GitHub Action by adding a file named `.github/workflows/atlas-ci.yaml` to your repo.
+5. Based on the type of database you are using, copy the following code into the workflow definition file. Set up
+the `ariga-token` input parameter to the secret name you chose in the previous step, and ensure your
+mainline branch and migration directory path are configured correctly:
+
+<Tabs
+defaultValue="mysql"
+values={[
+{label: 'MySQL', value: 'mysql'},
+{label: 'PostgreSQL', value: 'postgres'},
+{label: 'MariaDB', value: 'maria'},
+]}>
+<TabItem value="mysql">
+
+```yaml {7,11,36,39}
```

Comment:
The "new" way to highlight lines is by using comments in the `// hightlight-start` and `// hightlight-end` format. They won't be visible to the user and will not appear in the copied code when using the copy button. 

I think they are more clear to read and they "grow" with the snippet if you edit it.